lib/cylc/network/https/suite_command_server.py

```diff
@@ -148,13 +148,14 @@ def nudge(self):
 
     @cherrypy.expose
     @cherrypy.tools.json_out()
-    def insert_tasks(self, items, stop_point_string=None):
+    def insert_tasks(self, items, stop_point_string=None, no_check=False):
         if not isinstance(items, list):
             items = [items]
         if stop_point_string == "None":
             stop_point_string = None
         return self._put("insert_tasks", (items,),
-                         {"stop_point_string": stop_point_string})
+                         {"stop_point_string": stop_point_string,
+                          "no_check": True if no_check == 'True' else False})
```

Comment:
Objects serialised for transmission on the comms layer? See line 154 in the diff above where `stop_point_string` is treated in a similar way.